bundles/org.openhab.binding.tado/src/main/java/org/openhab/binding/tado/internal/handler/TadoHomeHandler.java

```diff
@@ -156,7 +168,24 @@ private synchronized void initializeBridgeStatusAndPropertiesIfOffline() {
                     return;
                 }
 
+                /*
+                 * If there is only one home, or if there is no valid configuration.homeId entry, then use the first
+                 * home id. Otherwise use the configuration.homeId entry value (if one exists). If there is no valid
+                 * configuration.homeId entry but there are multiple homes then log the available home to help the
+                 * user set up the proper configuration.homeId entry.
+                 */
                 Integer firstHomeId = homes.get(0).getId();
+                if (homes.size() > 1) {
+                    Integer configHomeId = configuration.homeId;
+                    if (configHomeId == null || configHomeId == 0) {
+                        logger.info("Trying first Home Id in the list [{}]", homes.stream()
```

Comment:
Why would the user want to configure that as the first is already automatically picked and the bindings continues to operate?

To be helpfull, it might be better to log _all_ available homes and set the thing offline ?

Maybe also adjust the logged text: to amke clear the home id was not set and it continue's with id xxx until the id is set in the configuration ?

Reply:
PS see the code comment above for more detailed explanation :)